src/snowflake/cli/_plugins/workspace/commands.py

```diff
@@ -224,3 +224,31 @@ def validate(
         entity_id,
         EntityActions.VALIDATE,
     )
+
+
+version = SnowTyperFactory(
+    name="version",
+    help="Manages versions for project entities.",
```

Comment:
```suggestion
    help="Manages versions for application package entities.",
```
or something similar

Reply:
this command group isn't specific to just application packages though, we could have other entity types that would support version operations

Reply:
Right, my bad. I'm not sure what "project entities" exactly means, but it probably doesn't really matter as we'll soon move the help messages to the concrete actions.